docs/tutorials/variational_intro.ipynb

```diff
@@ -0,0 +1,1625 @@
7iitdwcFpAkWQzDCPg7Aj722GN6/vnntWPHjmbfLy0tVXx8vN566y394he/kFQfroYMGaKVK1dq7Nixx/0ZLpdLTqdTpaWlioqKatf6AcAMWwtdmvrklwqz2/TNvHMUbAu4/08Gjqst399d4jegtLRUMTExx3w/KytLtbW1mjx5sm9denq6evfurZUrVza7jdvtlsvlarIAQFcyKCFS0WHBqqzx6Nv8UrPLAUwX8KEoNzdXTz/9tG644YZjtiksLJTdbld0dHST9YmJiSosLGx2m/nz58vpdPqW1NTU9iwbAExntVp0WsO4otU76EID/CYU3X333bJYLC0uW7dubbJNfn6+pk6dqksuuUTXXXddu9Yzd+5clZaW+pY9e/a06+cDgD/ISIuVJK3acdDkSgDzBZldQKM5c+Zo5syZLbZJS0vzPS8oKNBZZ52lcePG6aWXXmpxu6SkJNXU1KikpKTJ2aKioiIlJSU1u43D4ZDD4Wh1/QAQiDL6NVyBtuuQ6jxeBTGuCN2Y34Si+Ph4xcfHt6ptfn6+zjrrLI0aNUqvvPKKrNaWf4lHjRql4OBgLVmyRNOmTZMk5eTkKC8vT5mZmT+5dgAIVEOSoxQZEqSy6jptLnBpRGq02SUBpgm4/yXIz8/XxIkT1bt3b/35z3/W/v37VVhY2GRsUH5+vtLT07VmzRpJktPp1KxZszR79mwtW7ZMWVlZuuaaa5SZmdmqK88AoKuyWS2+s0Wrd9KFhu7Nb84UtdaiRYuUm5ur3Nxc9erVq8l7jbML1NbWKicnR5WVlb73nnjiCVmtVk2bNk1ut1tTpkzRc88916m1A4A/yugXq8VbirV6xyFdf0Z/s8sBTNMl5inqDMxTBKCr2ri3ROc/s0KRIUHKvv8c2awWs0sC2k23m6cIAHDiTkqOUoSjflzRln3MyYbui1AEAN1ckM2q0Q23/Fi9k/mK0H0RigAAyuhXP1/RauYrQjdGKAIAKCOt/gq0NbsOyetlqCm6J0IRAEDDejoVZreppLJWOUVlZpcDmIJQBABQsM2qUX0axhXRhYZuilAEAJAkjW24DxqDrdFdEYoAAJJ+uA/amp2HxBR26I4IRQAASdLwXtEKCbbqYEWNthWXm10O0OkIRQAASZI9yKoxfevPFq3IPWByNUDnIxQBAHzG9Y+TJK3IZbA1uh9CEQDAZ8KA+lC0esdB1Xm8JlcDdC5CEQDA56SUKDlDg1XmrtPG/FKzywE6FaEIAOBjs1o0rn/9pfkrtjGuCN0LoQgA0MS4hi60FdsJReheCEUAgCbGN5wpWr+7RFU1HpOrAToPoQgA0ES/uHClOENU4/Fq7S5mt0b3QSgCADRhsVg0vqEL7ctt+02uBug8hCIAwFHOGBQvSfrie8YVofsgFAEAjjJhQJwsFimnqEz7SqvMLgfoFIQiAMBReoTbNaJXtCTpS84WoZsgFAEAmtXYhfb594wrQvdAKAIANOvMhlD05bb93PID3QKhCADQrBG9nIoKCZKruk7f7OWWH+j6CEUAgGYF2aw6fSBdaOg+CEUAgGNq7EJbnlNsciVAxyMUAQCOaeLg+lC0cW+pil3VJlcDdCxCEQDgmBKiQjS8l1OStIyzRejiCEUAgBZNSk+UJC3ZQihC10YoAgC0aNKQBEnSl9sOqLrWY3I1QMchFAEAWnRySpSSokJUVevRqh0HzS4H6DCEIgBAiywWi85uOFtEFxq6MkIRAOC4JqU3hqIiGYZhcjVAxyAUAQCOa/yAOIUG21RQWq3NBS6zywE6BKEIAHBcIcE2nZVeP2fRvzftM7kaoGMQigAArTLl5CRJ0mebCk2uBOgYhCIAQKucnZ4gu82q7fsrtK2ozOxygHZHKAIAtEpkSLAmDIyTxNkidE2EIgBAq00dWt+F9m9CEbogQhEAoNV+NiRRNqtF3+1zaffBCrPLAdoVoQgA0Go9wu3KTIuVJC3cyFVo6FoIRQCANjl/RIok6aPsApMrAdoXoQgA0CZThibJbrMqp6hMWwuZyBFdB6EIANAmztBg30SO/+JsEboQQhEAoM3OH9FTUn0XGvdCQ1cRcKFo165dmjVrlvr166fQ0FD1799f8+bNU01NTYvbTZw4URaLpcly4403dlLVANC1TBqSoHC7TfklVVqfd9jscoB2EWR2AW21detWeb1evfjiixowYIA2bdqk6667ThUVFfrzn//c4rbXXXedHnzwQd/rsLCwji4XALqkkGCbpgxN0vvr8/XP9fka1SfG7JKAnyzgQtHUqVM1depU3+u0tDTl5OTo+eefP24oCgsLU1JSUkeXCADdwi9O7aX31+fr4+wC3XfeSQq128wuCfhJAq77rDmlpaWKiTn+/6W8+eabiouL09ChQzV37lxVVlYes63b7ZbL5WqyAAB+MDYtVr16hKrMXafPNjNnEQJfwIei3NxcPf3007rhhhtabHfllVfqjTfe0LJlyzR37ly9/vrruuqqq47Zfv78+XI6nb4lNTW1vUsHgIBmtVp0yaj6v41/X7vX5GqAn85i+MllA3fffbceeeSRFtts2bJF6enpvtf5+fk688wzNXHiRP31r39t089bunSpJk2apNzcXPXv3/+o991ut9xut++1y+VSamqqSktLFRUV1aafBQBd1d7DlTr90WUyDOmLO89S71jGasK/uFwuOZ3OVn1/+82Yojlz5mjmzJkttklLS/M9Lygo0FlnnaVx48bppZdeavPPy8jIkKRjhiKHwyGHw9HmzwWA7qRXjzBNGBCnL7cd0D+y9mj2OYPNLgk4YX4TiuLj4xUfH9+qtvn5+TrrrLM0atQovfLKK7Ja294LmJ2dLUlKTk5u87YAgB9cOjpVX247oHfX7dGtkwYq2BbwIzPQTQXcv9z8/HxNnDhRvXv31p///Gft379fhYWFKiwsbNImPT1da9askSRt375dDz30kLKysrRr1y599NFHmjFjhs444wwNHz7crF0BgC5hyslJiotwq
```

Comment:
To keep correcting the mistake this line should be replaced by:

"Thus, we can combine the notion that we aim at minimizing $E_{\\theta + \\delta \\theta}$ and traveling distance  $d^2 = \epsilon$ to obtain the Lagrangian that we want to minimize with respect to the parameter variations $\\delta \\theta$ ( $\\lambda$ is the Lagrange Multiplier):\n"